docs/UserGuide.md

```diff
@@ -217,40 +220,74 @@ Format: `user_guide`
 
 Format: `help` or `help [COMMAND]`
 
+* The `COMMAND` refers to any valid command that is implemented.
+
 Examples:
-* `help` Shows the list of commands.
+* `help` Shows the list of valid commands.
 * `help find` Shows the description of `find` command.
 * `help add` Shows the description of `add` command.
 
-Examples:
-* `find Mac` returns `MacDonald`
-* `find fairprice` returns `NTUC Fairprice` <br>
-
 ### Viewing a client : `view`
 
-`view` displays the remarks and transaction log of the specified client.
+Displays the remarks and transactions of the specified client.
```

Comment:
Should we also write that it is intended that the client panel is filtered? Maybe we can give the reason of removing noises from other data when we only want to focus on a specific client